File: 백준/Bronze/2480. 주사위 세개/주사위 세개.js
const fs = require('fs');
const [a, b, c] = fs.readFileSync('/dev/stdin').toString().trim().split(' ').map(Number);
let price;

if (a === b && b === c) {
    price = 10_000 + (a * 1_000);
} else if (a === b && b !== c) {
    price = 1_000 + (a * 100);
}  else if (b === c && c !== a) {
    price = 1_000 + (b * 100);
} else if (c === a && a !== b) {
    price = 1_000 + (c * 100);
} else {
    price = Math.max(a, b, c) * 100;
}

console.log(price);

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
# 같은 눈이 3개가 나오면 10,000원+(같은 눈)×1,000원의 상금을 받게 된다.
# 같은 눈이 2개만 나오는 경우에는 1,000원+(같은 눈)×100원의 상금을 받게 된다.
# 모두 다른 눈이 나오는 경우에는 (그 중 가장 큰 눈)×100원의 상금을 받게 된다.

A, B, C = map(int, input().split())

if A == B and B == C:
    print(A * 1_000 + 10_000)
elif A != B and B != C and C != A:
    print(max(A, B, C) * 100)
else:
    if A == B or A == C:
        print(A * 100 + 1_000)
    else:
        print(B * 100 + 1_000)